src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,149 @@
+
+### Keyboard interaction
+Filter panel includes a Skip to main content option for keyboard users so they can jump into the main content and/or search results.
+
+### Smaller breakpoints (md and sm)
+On smaller breakpoints, the filter panel is triggered by a button that can be configured anywhere within the experience. Once the user selects the filter button, a modal will open and display the filter panel, the user can close the modal to view the filtered results. The filter button should indicate how many filters the user has selected.
+
+*placeholder image*
+
+
+## Category guidance
+
+When determining the default display for your user, consider the information you have in your filter, including:
+
+* The ordering of categories
+* The number of categories
+* The number of items within each category
+
+Then decide which categories are open by default and which categories are closed. Try to strike a balance between what the user may need and the cognitive load of seeing all options at once.
+
+### Ordering of categories and items
+The ordering of top level categories is usually determined by relevance to user goals.
+
+Within the categories, we recommend the default ordering of items to be alphabetical unless business goals require a different ordering.
+
+### Number of categories
+
+#### One category
+If there is only one category, consider displaying all the filters open.
+
+*placeholder image*
+
+#### Two to nine categories
+If there are between two and nine categories of filters, consider having the top one to three categories open by default, and the other categories collapsed.
+
+*placeholder image*
+
+#### Ten or more categories
+If there are over ten categories, consider collapsing all of the categories on initial load, or having the first category open.
+
+*placeholder image*
+
+### Number of filter items within a category
+
+#### One to seven filter items
+If there are between one and seven filter items within a category, expose all of the items.
+
+*placeholder image*
+
+#### Eight or more filter items
+If there are eight or more filter items within a given category, consider utilizing the “View all” button. Expose a portion of the filter items so the user can quickly scan the options, once they select the view all button, the rest of the items are revealed. 
+
+*placeholder image*
+
+
+## Design and functional specifications
+
+
+For more details on Filter panel, see <a href="link to design specs" target="https://ibm.box.com/s/19a0hjn61gh3qh59kbgxba43y8uymuqy" rel="noreferrer">design specs</a> and see <a href="link to functional specs" target="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Filter-panel" rel="noreferrer">functional specs</a>.
+
+ <a href="Link to the BOX folder" target="_blank" rel="noreferrer">here</a>.
+
+
+## Development documentation
+
+<ComponentStatus name="Filter panel" type="ui" />
+
+## Feedback
+
+<ComponentFeedback />
```

Comment:
It looks like you began working on this before we updated each component with its footer.

```suggestion
<ComponentFooter name="Back to top" type="ui" />
```